update_repos.sh

```diff
@@ -229,11 +229,25 @@ main () {
         fi
         git commit -a -m 'release: v1.0.0' --author='Invenio <info@inveniosoftware.org>' --no-gpg-sign &> /dev/null
 
+        if [ -d "$HOME/$TEMPDIR/$repo/docs" ]; then
+            # Skip last 4 lines
+            head -n 4 LICENSE > LICENSE2
```

Comment:
Can you integrate this into the previous license change, so that the license change is not split over two commits.